plugins/repository-azure/src/main/plugin-metadata/plugin-security.policy

```diff
@@ -14,4 +14,6 @@ grant {
   // Used by jackson bean deserialization
   permission java.lang.RuntimePermission "accessDeclaredMembers";
   permission java.lang.reflect.ReflectPermission "suppressAccessChecks";
+  // needed by netty dns resolver even though we don't use it (!)
+  permission java.io.FilePermission "/etc/hosts", "read";
```

Comment:
I think we need to add permissions to list and read files within `/etc/resolver`. See https://github.com/netty/netty/blob/d34212439068091bcec29a8fad4df82f0a82c638/resolver-dns/src/main/java/io/netty/resolver/dns/UnixResolverDnsServerAddressStreamProvider.java#L73

Reply:
I added the persmissions and opened a followup (https://github.com/elastic/elasticsearch/issues/73479) to see how we might avoid using the netty dns resolver altogether.